src/Analyzers/CSharp/Analyzers/UseIndexOrRangeOperator/MemberInfo.cs

```diff
@@ -6,26 +6,20 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.UseIndexOrRangeOperator
 {
-    internal readonly struct MemberInfo
+    internal readonly struct MemberInfo(
```

Comment:
Why can't we use records here? The concerns I can see are assembly size and difficulty for documentation.

Reply:
> Why can't we use records here?

No one said we couldn't use records here :-)